modules/_labs/color-picker/react/lib/parts/SwatchBook.tsx

```diff
@@ -67,15 +67,9 @@ export const SwatchBook = ({colors, value, onSelect}: SwatchBookProps) => (
       const handleKeyDown = (event: React.KeyboardEvent) =>
         (event.key === 'Enter' || event.key === ' ') && onSelect(color);
 
-      const formatHex = (value: string) => {
-        return value.replace(/#/g, '').substring(0, 6);
-      };
-      const formattedColor = formatHex(color);
-
       return (
         <SwatchContainer
           key={index + '-' + color}
-          className={`wdc-color-picker--color-${formattedColor}`}
```

Comment:
This was treated as sort of a `data-testid` so I'm removing it here. There's no css associated with this class name. Hence, the cleaning up of Cypress tests in this PR rather than #831